distribution/tools/server-cli/src/main/java/org/elasticsearch/server/cli/APMJvmOptions.java

```diff
@@ -274,7 +234,7 @@ static Map<String, String> extractApmSettings(Settings settings) throws UserExce
             if (propertiesMap.containsKey(key)) {
                 throw new UserException(
                     ExitCodes.CONFIG,
-                    "Do not set a value for [tracing.apm.agent." + key + "], as this is configured automatically by Elasticsearch"
+                    "Do not set a value for [telemetry.agent." + key + "], as this is configured automatically by Elasticsearch"
```

Comment:
Just double checking... the old message made no mention of the `telemetry` prefix. I suppose it was incomplete before?

Reply:
yes, that message was previously missed 👍